packages/framework/src/Resources/views/Admin/Content/Superadmin/cssDocumentation.html.twig

```diff
@@ -1074,15 +1074,15 @@
                     {{ 'Textarea may be across the whole width'|trans }}
                 </label>
                 <div class="form-line__line">
-                    <textarea class="input input--textarea-big">{{ 'Textarea spread to the whole width. In this case it is not necessary to use form-line__item, because the element is across 100 % of width.'|trans }}</textarea>
+                    <textarea class="input input--textarea">{{ 'Textarea spread to the whole width. In this case it is not necessary to use form-line__item, because the element is across 100 % of width.'|trans }}</textarea>
```

Comment:
This and textarea on line 1085 seems same

Reply:
Also, there is textarea on line 784, that might be different because of the label, is it right? I do not know where to open this documentation live, so I could compare it rendered.

Reply:
The line 1085 and 1077 are not the same, the wrap is different, i dont actually want to change anything in this file, because it is used as showcase and i think that it should be reviewed as whole. I wouldnt spend time on this right now if you agree.

You can see documentation if you log into administration as superadmin, that way you can click on superadmin section where is documentation situated.

Reply:
Ok, just create US for that review.